proposals/apple-mdm/base-setup-clix.md

```diff
@@ -0,0 +1,139 @@
+# MDM Base Setup @ Dogfood
+
+## Items of Note
+- A NO-OP command (setup apple-mdm) which are pure instruction gathering only
+- Setup integrated into fleetctl, not namespaced into a separate vertical (e.g. `fleetctl apple-mdm`). That might happen at other parts of the UX, but not for setup.
+
+## fleetctl setup
+
+```
+$ fleetctl setup
+NAME:
+   fleetctl setup - Set up a Fleet instance or MDM
+
+USAGE:
+   fleetctl setup [options]
+   fleetctl setup apple-mdm
+   fleetctl setup apns
+   fleetctl setup scep
```

Comment:
We are missing a `setup dep` step. See https://github.com/fleetdm/fleet/blob/apple-mdm-proposal/proposals/apple-mdm/mvp-dogfood/setup.md#dep.